views/use_the_data.erb

```diff
@@ -0,0 +1,256 @@
+<div class="container" id="contribute">
+  <div class="page-section">
+    <div class="row">
+      <div class="column-one-quarter">
+         <%= erb :about_nav_menu %>
+      </div>
+      <div class="column-three-quarters">
+        <h1>Use EveryPolitician data</h1>
+
+        <p class="section-intro">
+			We gather together all this lovely data, curate it, and make it
+			reliably available... so you can use it. Here&rsquo;s the bare bones of how
+			your application can get it.
+        </p>
+        <p>
+			This page is for you if you&rsquo;re a developer who is planning on
+			building something wonderful or brilliant with the data we have on your
+			country&rsquo;s politicians. It tells you how to get the data and, broadly,
+			how it&rsquo;s structured (spoiler: CSV or Popolo JSON).
+        </p>
+		<p>
+			We believe that by making this data available in consistent and
+			useful formats, if you write something neat that uses, for example, the data
+			for your country&rsquo;s politicians, and open it up under a free software open
+			source license... then other people will be able to adapt what you&rsquo;ve
```

Comment:
Yeah it's clumsy this... problem is that just making a tool that works with the data doesn't mean anyone else can use it. Gah.